gtfs-realtime/CHANGES.md

```diff
@@ -18,7 +18,7 @@ The official specification, reference and documentation are written in English.
   - Anyone is allowed to vote yes/no in a form of comment to the pull request, and votes can be changed until the end of the voting period.
     If a voter changes her vote, it is recommended to do it by updating the original vote comment by striking through the vote and writing the new vote.
   - Votes before the start of the voting period are not considered.
-1. The proposal is accepted if there is a consensus yes with at least 3 votes.
+1. The proposal is accepted if there is a unanimous consensus yes with at least 3 votes.
```

Comment:
Actually, the article is for "consensus", not "unanimous", so "a" is
correct.
On Jul 29, 2015 9:36 PM, "Ivan Egorov" <notifications@github.com> wrote:

> In gtfs-realtime/CHANGES.md
> <https://github.com/google/transit/pull/11#discussion_r35801992>:
>
> > @@ -18,7 +18,7 @@ The official specification, reference and documentation are written in English.
> >    - Anyone is allowed to vote yes/no in a form of comment to the pull request, and votes can be changed until the end of the voting period.
> >      If a voter changes her vote, it is recommended to do it by updating the original vote comment by striking through the vote and writing the new vote.
> >    - Votes before the start of the voting period are not considered.
> > -1. The proposal is accepted if there is a consensus yes with at least 3 votes.
> > +1. The proposal is accepted if there is a unanimous consensus yes with at least 3 votes.
>
> either "an unanumous" or just drop the article altogether?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/google/transit/pull/11/files#r35801992>.
>